superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -354,39 +355,28 @@ const SqlEditor = ({
     return base;
   }, [dispatch, queryEditor.sql, startQuery, stopQuery]);
 
-  const handleWindowResize = useCallback(() => {
+  const handleWindowResize = useEffectEvent(() => {
```

Comment:
You should pass the `height` to the effect event. Check the **Notes** [here](https://react.dev/learn/separating-events-from-effects).

```suggestion
  const handleWindowResize = useEffectEvent(height => {
```

Reply:
removed `handleWindowResize` since no longer needed